server/controllers/article.controller.js

```diff
@@ -158,13 +158,65 @@ class ArticleController {
       },
       {
         model: Comment,
-        as: 'comments'
+        as: 'articlecomments',
       }]
     });
     if (articles.count === 0) {
       return Response(res, 400, 'There is no article in database');
     }
     return Response(res, 200, 'Articles successfully retrieved', { articles });
   }
+
+  /**
+    * @description gets a single article
+    * @param {object} req
+    * @param {object} res
+    * @returns {object} An article and it's associated comments
+    */
+  static async getOneArticle(req, res) {
+    const { slug: articleSlug } = req.params;
+    // const articleSlug = req.params.slug;
+    const isArticleExists = await Article.findOne({
```

Comment:
This variable naming can be better `doesArticleExists`

Reply:
noted